src/cargo/util/command_prelude.rs

```diff
@@ -100,11 +100,11 @@ pub trait AppExt: Sized {
     }
 
     fn arg_features(self) -> Self {
-        self._arg(
-            opt("features", "Space-separated list of features to activate")
-                .multiple(true)
-                .value_name("FEATURES"),
-        )
+        self._arg(optional_multi_opt(
```

Comment:
Why is this `optional_multi_opt` and not just `multi_opt`?  This change will make an empty features flag valid (`cargo run --features`).

Reply:
Oops, good point. Misread the options!